nikolay.braslavets/hw01/2.rb

```diff
@@ -0,0 +1,4 @@
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234',
+         yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+
+p hash.count { |k,_| k =~ /yes/i }
```

Comment:
That is really cool that you used count with a block without any help. But sadly this code is not valid, we need to count amount of keys that starts with 'yes' and not the count of keys that include yes. 
![image](https://user-images.githubusercontent.com/16926726/97117868-248b7680-170f-11eb-9916-5099097a4a91.png)